src/features/commands.ts

```diff
@@ -567,6 +567,53 @@ Here's an article explaining the difference between the two: https://goshakkk.na
       });
     },
   },
+  {
+    words: [`!jwt`],
+    help: `Describes reasoning for and against the use of JWT tokens againt using standard sessions`,
+    category: "Reactiflux",
+    handleMessage: (msg) => {
+      msg.channel.send({
+        embeds: [
+          {
+            title:
+              "Is JWT the right approach for my applications authentication?",
+            type: "rich",
+            description: `
+            **TL;DR: JWT is widely overused as the go to authentication choice over sessions. Use this bot to form your opinon on wether it is a good fit for you.**
+            
+            A lot of developers starting their journey into programming will, at some point, hit the question of how to work with backend authentication.
+            They will undoubtably come across the multitude of JWT tutorials on the subject. But is this the correct approach?. Most of the time the answer will be no. 
+            So what is the alternaive? Sessions. Sessions have been tried and tested for decades, with a lot of back end frameworks supporting them out of the box.
+            They take up miminal setup times, are completely managed by the server and cannot, if correctly set up , be accessed by the client
```

Comment:
Redundant space before comma in "up , be".
Possibly lack of terminal dot at the end of sentence "by the client".